src/main/python/MANIFEST.in

```diff
@@ -3,3 +3,4 @@ include LICENSE.txt
 
 # documentation, if any - there may be a minimal set of docs:
 # recursive-include docs *.html *.txt *.js
+recursive-include deps/jars *.jar
```

Comment:
when does the jar get built in your workflow above?

Reply:
I had cheated and uploaded it into a commit. Updated  script to the pipeline so it will pull the jar from there.